app/models/degree.rb

```diff
@@ -7,11 +7,36 @@ class Degree < ApplicationRecord
   validates :non_uk_degree, presence: true, on: :non_uk
   validates :subject, presence: true, on: %i[uk non_uk]
   validates :institution, presence: true, on: :uk
-  validates :graduation_year, presence: true, on: %i[uk non_uk],
-                              inclusion: { in: 1900..Time.zone.today.year }
+  validates :graduation_year, presence: true, on: %i[uk non_uk]
+  validate :future_year
+  validate :valid_year
   validates :grade, presence: true, on: :uk
 
   belongs_to :trainee
 
   enum locale_code: { uk: 0, non_uk: 1 }
+
+  def future_year
+    if graduation_years.present? && graduation_years > last_valid_year
+      errors.add(:graduation_year, :future)
+    end
+  end
+
+  def valid_year
+    first_year = last_valid_year - 50
```

Comment:
It's unlikely, but I think this would preclude someone in their 70s from being registered to teach?

An existing teacher moving to the uk _could_ fall in this bracket if doing assessment only.

Reply:
I've increased the limit by ten years, shouldn't be anyone in their 80s